mkdocs.yml

```diff
@@ -12,7 +12,7 @@ google_analytics:
   - 'auto'
 strict: true
 theme:
-  name: 'material'
+  name: 'mkdocs'
```

Comment:
Did you mean to do this? This breaks the site.

Reply:
my bad - mistake to be corrected.